modules/runner/src/main/ts/runner/TestRun.ts

```diff
@@ -92,6 +106,23 @@ export const runTests = (tests: Test[], state: RunState, actions: RunActions, re
   return loop(tests, (test) => runTest(test, state, actions, reporter));
 };
 
+const fakeFailure = (reporter: RunReporter, suite: Suite, state: RunState, siblings: string[], actions: RunActions) => {
+  const tests = suite.tests.length === 0 ? suite.suites[0].tests : suite.tests;
+  const filename = tests[0]?.file || 'Unknown';
+  const fakeReporter = reporter.test(filename.substring(filename.lastIndexOf('/') + 1) + ' DOM validation', suite.fullTitle(), state.totalTests);
+  fakeReporter.start();
+  const fakeFailure = LoggedError.loggedError(new Error(`File ${filename} did not clean up after itself, extra elements were left in the DOM`), siblings);
+
+  // This is mostly duplicate of `fakeReporter.fail()`, but the "test" doesn't really exist
+  // so the UI gets confused and shows two errors if we do that
+  console.error(fakeFailure);
+  fakeReporter.fail(fakeFailure);
+  // this is where the page reloads if the global retry system is active
+  actions.onFailure();
+  // Test failures must be an empty reject, otherwise the error management thinks it's a bedrock error
+  return Promise.reject();
```

Comment:
Hmm, I think you should use subclasses or Error if you want to distinct on error type here if possible since this seems fragile.